examples/README.md

```diff
@@ -2,26 +2,34 @@
 
 This directory contains a collection of ruby example to help get you started using the Lob API. As always, feel free to [contact us](https://lob.com/support) directly if you have any questions.
 
+## /csv_checks/
+
+An example showing how to dynamically create checks from a CSV using HTML, variable data, and Lob's [Checks API](https://lob.com/services/checks).
+
+## /csv_letters/
+
+An example showing how to dynamically create letters from a CSV using HTML, a custom font, variable data, and Lob's [Letters API](https://lob.com/services/letters).
+
 ## /csv_postcards/
 
-An example showing how to dynamically create postcards from a CSV using HTML, a custom font, variable data, and Lob's [Simple Postcard Service](https://lob.com/services/postcards).
+An example showing how to dynamically create postcards from a CSV using HTML, a custom font, variable data, and Lob's [Postcards API](https://lob.com/services/postcards).
```

Comment:
👍 good catch. Would you mind checking the other wrappers and making sure they no longer reference the "Simple _______ Service"?